### PR TITLE
Build landing page and it's corresponding side views

### PR DESCRIPTION
**Pull request by Irirwanirira**
#### What does this PR do?

Add Welcome, Holla, and Find Parking Screen view to the Hoop app

#### Description of the task to be completed

This pull request adds three new screens to the mobile app: Welcome Screen, Holla Screen, and Find Parking Screen. Each screen includes the necessary static content such as links to login, sign up, skip button, and appropriate images and icons.

**Changes:**
- Added Welcome Screen with static content and links.
- Added Holla Screen with static content and links.
- Added Find Parking Screen with static content and links.
- Added necessary images and icons to each screen.

**Testing Done:**
- Tested each screen on a device screen to ensure proper rendering.
- Checked alignment and styling across different platforms.

**Checklist:**
- [x] Static content is consistent with the design specifications.
- [x] Images and icons are correctly displayed on all devices.
- [X] Code follows the project's coding standards and conventions.

#### How should this manually be tested?

These changes could be tested by running the app on your Android Studio emulator or by having the Expo app installed on your phone and scanning the QR code to try the app.

#### What are the relevant pivotal tracker/Trello stories?

[Trello card](https://trello.com/c/gtZDjQSg)

### CODE REVIEW

File: app/index.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import ExploreScreen from './screens/ExploreScreen';
+import HistoryScreen from './screens/HistoryScreen';
+import HomeView from './screens/HomeView';
+import ProfileScreen from './screens/Home/ProfileScreen';
+import DetailCategoryScreen from './screens/Home/DetailCategoryScreen';
+import ParkingDetailsScreen from './screens/Home/ParkingDetails';
+
+const HomeScreen = () => {
+    const [isReady, setIsReady] = useState(false)
+    // const [fontLoaded, fontError] = useFonts({
```
Comment (23nosurrend) [inline app/index.tsx:21]: Remove these comments

Comment (KeynesBizimana) [inline app/index.tsx]: Remove these comments

File: app/index.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    //     setIsMounted(true)
+    // },[])
+
+    // useEffect(()=> {
+    //     if(isMounted){
+    //         setTimeout(()=> {
+    //             router.navigate('./screens/WelcomeScreen')
+    //         }, 1000)
+    //     }
+    // }, [isMounted])
```
Comment (KeynesBizimana) [inline app/index.tsx]: Remove these comments

**Review by KeynesBizimana — CHANGES_REQUESTED** [review]
It is good so far, but remove these comments

**Review by KeynesBizimana — APPROVED** [review]
Alright

**Pull request merged**